cmake/platform/linux/webos.cmake

```diff
@@ -0,0 +1,8 @@
+include(${CMAKE_SOURCE_DIR}/cmake/platform/${CORE_SYSTEM_NAME}/wayland.cmake)
+
+# add wayland as platform, as we require it.
+# saves reworking other assumptions for linux windowing as the platform name.
+list(APPEND CORE_PLATFORM_NAME_LC wayland)
+
+list(APPEND PLATFORM_REQUIRED_DEPS WaylandProtocolsWebOS)
+list(APPEND PLATFORM_GLOBAL_TARGET_DEPS generate-wayland-webos-protocols)
```

Comment:
Add the following in here

```
list(APPEND ARCH_DEFINES -DTARGET_WEBOS)
```

Not sure if TARGET_LINUX_WEBOS is "better" to explicitly mention the parent, but id personally just go with the above.